pkg/asset/internal/templates.go

```diff
@@ -282,9 +282,29 @@ metadata:
   labels:
     k8s-app: kube-controller-manager
 spec:
-  replicas: 2
+  replicas: 1
   template:
     metadata:
+      annotations:
```

Comment:
You mean use a function to set the annotation? I think it's ok to have some duplications in the template.